apps/native/app/SignUp.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { ThemedView } from "@/components/ThemedView";
+import { ThemedText } from "@/components/ThemedText";
+import { Button } from "@/components/Button";
+import { StyleSheet, TextInput, View } from "react-native";
+import { SafeAreaView, SafeAreaProvider } from "react-native-safe-area-context";
+import { Text } from "react-native";
+
+const SignUp = () => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const Submit = () => {
+    //signup
+  };
+
+  const GmailSubmit = () => {
+    //signup with gmail
+  };
+
+  const FacebookSubmit = () => {
+    //signup with facebook
+  };
+
+  const GithubSubmit = () => {
+    //signup with github
+  };
+
+  return (
+    <SafeAreaProvider>
+      <SafeAreaView>
+        <Text>Sign Up</Text>
+        <TextInput
+          style={styles.input}
+          value={name}
+          onChange={e => setName}
+          placeholder="Name"></TextInput>
+        <TextInput
+          style={styles.input}
+          value={email}
+          onChange={e => setEmail}
+          placeholder="Email"></TextInput>
+        <TextInput
+          style={styles.input}
+          value={password}
+          onChange={e => setPassword}
+          secureTextEntry={true}
+          placeholder="Password"></TextInput>
```

Comment:
The <TextInput><TextInput/></TextInput> syntax is incorrect. Use <TextInput /> instead for proper JSX formatting.
To reduce code duplication, refactor repeated <TextInput> logic into a reusable component.